lib/index.js

```diff
@@ -51,8 +57,8 @@ class BaseAsset {
 
     get asset_id() { return this._asset_id; }
 
-    set_title(value) { this._title = value; }
-    set_synopsis(value) { this._synopsis = value; }
+    set_title(value) { this._title = _sanitize_text(value); }
```

Comment:
ok, I will move it inline then (for the record, I though that in the long term having trim/replace all scattered around wouldn't read too nicely).